test/test.js

```diff
@@ -294,6 +294,45 @@ test('Remove mockjax definition by id', function() {
 	});
 });
 
+test('Remove mockjax definition by url', function() {
+	$.mockjax({
```

Comment:
For completeness, let's add a test with a subpath as well (`'/test/foo'`) and make sure that once we remove the `/test` handler that the subpath still works